docs/OnTrack/Deployment/Enhanced Authentication/current-and-proposed-authentication-evaluation-5.1.md

```diff
@@ -0,0 +1,185 @@
+  the systems operate and what has occurred in the instance of an error or breach. Having an
+  independent authentication solution streamlines the control and medication of the services
+  provided by it, allowing for access and control to be granted solely to the OnTrack team on the
+  backend of the project. In this way, the LDAP and Devise features used can be customised to suit
+  the specific (and changing) needs of the OnTrack system, using only features that are necessary
+  and relevant to the OnTrack system and user authentication needs, rather than accepting all
+  services provided by third-party authentication mechanisms
+- As stated, the OnTrack team will be granted full control over the authentication systems and
+  elements of the OnTrack software. This will solve the issue of the redirection to the broken
+  Deakin SSO link as explored in previous sections, as the team will be able to control the flow
+  towards other pages after the successful signing out of a user.
+
+### Potential Issues and Concerns that must be Considered
+
+Within the development and deployment phases related to implementing these proposed authentication
+mechanisms, the following must be closely considered and addressed:
+
+- It must be ensured that, while system admins have the ability to grant access to change the
+  passwords, that they cannot directly access the passwords themselves. There must be a clear
+  separation of information, and passwords must have secure encryption applied to them
+- Password change requests must be validated to ensure that they are coming from the user, and not
+  someone pretending to be the user in order to gain control over the account. A security mechanism
+  such as requiring the requestee’s date of birth, or an answer to a security question set by the
+  user, is suggested. Additionally, sending an automatic email or text to inform the user of the
+  request for a change of password is also recommended
+- The developed authentication system, including how user information is stored, transmitted, and
+  later deleted (which includes considerations of retention laws) must adhere to the appropriate
+  laws and guidelines set out by the Australian Federal Government (including the _Privacy Act
+  1988_), as well as other specifications mandated within each of the Australian states and
+  territories
+- Additionally, as OnTrack provides its services to international users (both in terms of
+  international students studying online at Deakin and through providing its services to
+  international clients), OnTrack’s authentication and information storage processes must also
+  adhere to the laws enacted within the relevant international jurisdictions. For example, the data
+  protection laws enacted within the GDPR must be adhered to by companies who provide services to
+  citizens protected under the GDPR, regardless of where the company is situated (see
+  https://gdpr-info.eu/)
```

Comment:
I am so glad to see you including the GDPR! Really getting into the Global Citizenship portion of the GLOs